textproc/lilypond/Portfile

```diff
@@ -17,113 +16,96 @@ long_description    Lilypond is a unix-based automated engraving system that \
                     similar to LaTeX. Lilypond can export sheet music to PDF, \
                     EPS, SVG, and PNG formats, and can also create MIDI files.
 
-homepage            http://lilypond.org/
-platforms           darwin
-master_sites        http://download.linuxaudio.org/lilypond/sources/v${branch}/
-#dist_subdir         ${name}/${version}_${revision}
-checksums           rmd160  a355201334fd0beac36c96f844c75d8e27b4ea9d \
-                    sha256  329d733765b0ba7be1878ae3f457dbbb875cc2840d2b75af4afc48c9454fba07
+homepage            https://lilypond.org
+master_sites        https://lilypond.org/download/sources/v2.20/
+checksums           rmd160  030ebe2074ad647269c7f8aca40eaee671ddb77f \
+                    sha256  595901323fbc88d3039ca4bdbc2d8c5ce46b182edcb3ea9c0940eba849bba661 \
+                    size    18100533
 
+platforms           darwin
+universal_variant   no
 
-depends_build       port:bison \
+depends_build       port:autoconf \
+                    port:bison \
+                    port:dblatex \
+                    port:flex \
+                    port:fontforge \
+                    port:gmake \
+                    port:netpbm \
+                    port:p5.28-encode \
+                    port:p5.28-pod-escapes \
+                    port:p5.28-pod-simple \
+                    port:p5.28-podlators \
+                    port:p5.28-scalar-list-utils \
+                    port:pango \
+                    port:pkgconfig \
+                    port:rsync \
                     port:t1utils \
                     port:texi2html \
-                    port:pkgconfig
-depends_lib         port:fontconfig \
-                    port:fontforge \
-                    port:freetype \
-                    port:gettext \
-                    port:gmp \
-                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
-                    port:ghostscript \
-                    port:mftrace \
-                    port:guile18 \
                     port:texinfo \
-                    path:lib/pkgconfig/pango.pc:pango \
-                    port:flex \
-                    port:t1utils \
+                    port:texlive-fonts-recommended \
                     port:texlive-lang-cyrillic \
                     port:texlive-metapost \
-                    port:dblatex \
-                    port:urw-fonts \
-                    port:libtool \
-                    path:bin/Xquartz:xorg-server \
-                    port:python27
+                    port:texlive-xetex \
+                    port:urw-core35-fonts
 
-configure.python    ${frameworks_dir}/Python.framework/Versions/2.7/bin/python2.7
+depends_lib         port:extractpdfmark \
+                    port:ghostscript \
+                    port:guile18 \
+                    port:python27
```

Comment:
I suggest that you update this and relevant places to python37 (or 38)

Reply:
Ack. Currently testing to make sure build/run work as expected, will push update once verified.